doc/tutorial/index.rst

```diff
@@ -6,7 +6,10 @@ This is an introductory tutorial to **servant**. Whilst browsing is fine, it mak
 Any comments, issues or feedback about the tutorial can be submitted
 to `servant's issue tracker <http://github.com/haskell-servant/servant/issues>`_.
 
-In fact, the whole tutorial is a `cabal <https://cabal.readthedocs.io/en/latest/>`_
+Cabal
```

Comment:
Just to avoid any confusion for potential readers, I'd use `cabal-install` (the program, for which the executable is called `cabal`) here, since `Cabal is a library that is used by both `cabal-install` and `stack`.